packages/devtools-ui-kit/src/components/NDialog.vue

```diff
@@ -21,7 +21,7 @@ const emit = defineEmits<{
 const show = useVModel(props, 'modelValue', emit, { passive: true })
 const card = ref(null)
 
-const { activate, deactivate } = useFocusTrap(card, { immediate: false })
+const { activate, deactivate } = useFocusTrap(card.value, { immediate: false })
```

Comment:
This does not seem right, we are passing the ref on purpose as it will change reactively.

Reply:
@antfu when it was `useFocusTrap(card, { immediate: false })` i was getting this error:
`Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times`

Reply:
can we use `<Teleport v-lazy-show="show" to="body">` ?

Reply:
@antfu changed it to v-lazy-show, sorry in advance if i messit up,
but still there is a problem with the focusing on input for first time that opens


https://github.com/nuxt/devtools/assets/38922203/c40af9ea-c489-48d3-a1b1-c85dff12b2a3